client/src/Components/ReviewsContainer/List/ReviewTile.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from "react"
+import Card from "react-bootstrap/Card"
+import Container from "react-bootstrap/Container"
+import Row from "react-bootstrap/Row"
+import Col from "react-bootstrap/Col"
+import Image from "react-bootstrap/Image"
+import Modal from "react-bootstrap/Modal"
+import StarRatings from "react-star-ratings"
+import useReviews from "../../../contexts/ReviewsContext.jsx"
+
+const ReviewTile = (props) => {
+  const { markReviewHelpful, reportReview } = useReviews()
+  const [helpful, setHelpful] = useState(false)
+  const [report, setReport] = useState(false)
+
+  const header = () => (
+    <Container>
+      <Row>
+        <Col className="stars">
+          <StarRatings
+            className="stars"
+            rating={props.rating}
+            starDimension="15px"
+            starSpacing="0"
+          />
+        </Col>
+        <Col className="name-date">
+          {props.reviewer_name}
+          {", "}
+          {new Date(props.date).toLocaleDateString("en-US", {
+            year: "numeric",
+            month: "long",
+            day: "numeric",
+          })}
+        </Col>
+      </Row>
+    </Container>
+  )
+
+  const recommend = () => {
+    if (props.recommend) {
+      return (
+        <Card.Text className="recommend">
+          &#10003; I recommend this product!
+        </Card.Text>
+      )
+    }
+  }
+
+  const response = () => {
+    if (props.response) {
+      return (
+        <Card>
+          <Card.Title>Response</Card.Title>
+          <Card.Text>{props.response}</Card.Text>
+        </Card>
+      )
+    }
+  }
+
+  const photos = () => {
+    if (props.photos.length > 0) {
+      return (
+        <Container>
+          <Row>
+            {props.photos.map((photo) => {
```

Comment:
Maybe create another Component for this? May be practice to have a useState within a component.  Code is fine though 👍